lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/Constants.java

```diff
@@ -60,6 +64,8 @@
   public static final Configuration HTTP_SERVER_CONFIGURATION = Configurations.merge(
       HttpHandlerConfiguration.CONF
           .set(HttpHandlerConfiguration.HTTP_HANDLERS, HttpServerReefEventHandler.class)
+          // Add the http status handler.
+          .set(HttpHandlerConfiguration.HTTP_HANDLERS, DriverStatusHTTPHandler.class)
```

Comment:
You don't need to bind for DriverStatusHTTPHandler. This is for HttpServerReefEventHandler binding only. It is the unique one for entire server. For all the implementations of HttpHandler, REEF code will automatically detect all such handlers.